components/resourceDetails/index.jsx

```diff
@@ -93,8 +95,15 @@ export default function ResourceDetails({
     getAccordionKey(dataset, "sharing"),
     false
   );
-  const useAcp = isAcp(datasetUrl, fetch);
-  const useWac = isWac(datasetUrl, dataset, fetch);
+
+  useEffect(() => {
+    (async () => {
+      const acp = await isAcp(datasetUrl, fetch);
+      const wac = await isWac(datasetUrl, dataset, fetch);
+      setUseAcp(acp);
+      setUseWac(wac);
+    })();
+  }, [datasetUrl, dataset, fetch]);
```

Comment:
What is the difference between doing this and using the `useSWR` hook, and is one approach better than the other in this context ?

Reply:
We could have separate hooks (`useAcp`, `useWac`) for each value.